addon/components/search-result/component.js

```diff
@@ -50,6 +50,7 @@ export default Ember.Component.extend(Analytics, hostAppName, {
     * @property {Object} themeProvider
     */
     themeProvider: null,
+    domainRedirectProviders: null,
```

Comment:
It would be better to initialize this to an empty list to avoid exceptions if it's not passed in.